internal/controller/factory/builders.go

```diff
@@ -47,3 +48,42 @@ func reconcileSTS(ctx context.Context, rclient client.Client, crdName string, st
 	sts.Status = currentSts.Status
 	return rclient.Update(ctx, sts)
 }
+
+func reconcileConfigMap(ctx context.Context, rclient client.Client, crdName string, configMap *corev1.ConfigMap) error {
+	logger := log.FromContext(ctx)
+
+	currentConfigMap := &corev1.ConfigMap{}
+	err := rclient.Get(ctx, types.NamespacedName{Namespace: configMap.Namespace, Name: configMap.Name}, currentConfigMap)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			logger.V(2).Info("creating new configMap", "cm_name", configMap.Name, "crd_object", crdName)
+			return rclient.Create(ctx, configMap)
+		}
+		return fmt.Errorf("cannot get existing configMap: %s, for crd_object: %s, err: %w", configMap.Name, crdName, err)
+	}
+	configMap.Annotations = labels.Merge(currentConfigMap.Annotations, configMap.Annotations)
+	if configMap.ResourceVersion != "" {
+		configMap.ResourceVersion = currentConfigMap.ResourceVersion
+	}
+	return rclient.Update(ctx, configMap)
+}
+
+func reconcileSVC(ctx context.Context, rclient client.Client, crdName string, svc *corev1.Service) error {
```

Comment:
When I use camelCase names, usually I use them like this: reconcileSvc. Although I used it before the same: reconcileSVC. Why I changed my mind.
I google best practises and found one explanation.

Camel case and snake case should be able to be converted automatically to each other.

reconcileSVC will be converted to reconcile_s_v_c, because every capital letter in camelCase starts new word. And svc is one word. So in my opinion even abbreviations should not be written in capital letters.

This is not a blocking issue, just a comment to a code style, although we do not have common guide.